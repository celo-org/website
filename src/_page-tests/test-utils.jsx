import { render } from "@testing-library/react";
import * as React from "react";
import i18nForTests from "src/utils/i18nForTests";
import { I18nextProvider } from "react-i18next";
export function TestProvider({ children }) {
    return <I18nextProvider i18n={i18nForTests}>{children}</I18nextProvider>;
}
export const renderIgnoringUnstableFlushDiscreteUpdates = (component) => {
    const originalError = console.error;
    const error = jest.fn();
    console.error = error;
    const result = render(component);
    expect(error).toHaveBeenCalledTimes(1);
    expect(error).toHaveBeenCalledWith("Warning: unstable_flushDiscreteUpdates: Cannot flush updates when React is already rendering.%s", expect.any(String));
    console.error = originalError;
    return result;
};
//# sourceMappingURL=test-utils.jsx.map