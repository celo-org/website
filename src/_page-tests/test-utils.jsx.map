{"version":3,"file":"test-utils.jsx","sourceRoot":"","sources":["test-utils.tsx"],"names":[],"mappings":"AAAA,OAAO,EAAE,MAAM,EAAE,MAAM,wBAAwB,CAAA;AAC/C,OAAO,KAAK,KAAK,MAAM,OAAO,CAAA;AAC9B,OAAO,YAAY,MAAM,wBAAwB,CAAA;AAIjD,OAAO,EAAE,eAAe,EAAE,MAAM,eAAe,CAAA;AAI/C,MAAM,UAAU,YAAY,CAAC,EAAE,QAAQ,EAAS;IAC9C,OAAO,CAAC,eAAe,CAAC,IAAI,CAAC,CAAC,YAAY,CAAC,CAAC,CAAC,QAAQ,CAAC,EAAE,eAAe,CAAC,CAAA;AAC1E,CAAC;AAID,MAAM,CAAC,MAAM,0CAA0C,GAAG,CAAC,SAA6B,EAAE,EAAE;IAE1F,MAAM,aAAa,GAAG,OAAO,CAAC,KAAK,CAAA;IACnC,MAAM,KAAK,GAAG,IAAI,CAAC,EAAE,EAAE,CAAA;IACvB,OAAO,CAAC,KAAK,GAAG,KAAK,CAAA;IACrB,MAAM,MAAM,GAAG,MAAM,CAAC,SAAS,CAAC,CAAA;IAChC,MAAM,CAAC,KAAK,CAAC,CAAC,qBAAqB,CAAC,CAAC,CAAC,CAAA;IACtC,MAAM,CAAC,KAAK,CAAC,CAAC,oBAAoB,CAChC,iGAAiG,EACjG,MAAM,CAAC,GAAG,CAAC,MAAM,CAAC,CACnB,CAAA;IACD,OAAO,CAAC,KAAK,GAAG,aAAa,CAAA;IAE7B,OAAO,MAAM,CAAA;AACf,CAAC,CAAA"}