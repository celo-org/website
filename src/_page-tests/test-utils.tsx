import { render } from "@testing-library/react"
import * as React from "react"
import i18nForTests from "src/utils/i18nForTests"
interface Props {
  children?: React.ReactNode
}
import { I18nextProvider } from "react-i18next"



export function TestProvider({ children }: Props) {
  return <I18nextProvider i18n={i18nForTests}>{children}</I18nextProvider>
}

// https://github.com/testing-library/react-testing-library/issues/470
// an issue with muted on video causing a rerender, this seems to be be best solution for now
export const renderIgnoringUnstableFlushDiscreteUpdates = (component: React.ReactElement) => {
  // tslint:disable: no-console
  const originalError = console.error
  const error = jest.fn()
  console.error = error
  const result = render(component)
  expect(error).toHaveBeenCalledTimes(1)
  expect(error).toHaveBeenCalledWith(
    "Warning: unstable_flushDiscreteUpdates: Cannot flush updates when React is already rendering.%s",
    expect.any(String)
  )
  console.error = originalError
  // tslint:enable: no-console
  return result
}
