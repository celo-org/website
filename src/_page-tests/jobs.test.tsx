import JoinJobsPage from 'pages/jobs'
import * as React from 'react'
import * as renderer from 'react-test-renderer'
import { TestProvider } from 'src/_page-tests/test-utils'

describe('JoinJobsPage', () => {
  describe('with jobs', () => {
    it('renders', () => {
      const tree = renderer
        .create(
          <TestProvider>
            <JoinJobsPage />
          </TestProvider>
        )
        .toJSON()
      expect(tree).toMatchSnapshot()
    })
  })
  describe('without jobs', () => {
    it('renders', () => {
      const tree = renderer
        .create(
          <TestProvider>
            <JoinJobsPage />
          </TestProvider>
        )
        .toJSON()
      expect(tree).toMatchSnapshot()
    })
  })
})
