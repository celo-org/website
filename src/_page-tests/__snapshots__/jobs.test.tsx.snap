// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JoinJobsPage with jobs renders 1`] = `
.emotion-4 {
  opacity: 0;
}

.emotion-17 {
  width: 100%;
  height: 1rem;
}

<div
  className="emotion-0"
  style={
    Object {
      "WebkitAlignItems": "center",
      "WebkitBoxAlign": "center",
      "WebkitBoxPack": "center",
      "WebkitJustifyContent": "center",
      "alignItems": "center",
      "justifyContent": "center",
      "maxWidth": "100vw",
      "msFlexAlign": "center",
      "msFlexPack": "center",
      "overflowX": "hidden",
      "overflowY": "hidden",
    }
  }
>
  <div
    className="emotion-0"
    style={
      Object {
        "height": "100vh",
        "marginBottom": "100px",
        "width": "100%",
      }
    }
  >
    <div
      className="emotion-0"
      style={
        Object {
          "WebkitAlignItems": "center",
          "WebkitBoxAlign": "center",
          "WebkitBoxPack": "center",
          "WebkitJustifyContent": "center",
          "alignItems": "center",
          "height": "100vh",
          "justifyContent": "center",
          "left": "0px",
          "msFlexAlign": "center",
          "msFlexPack": "center",
          "position": "absolute",
          "right": "0px",
          "width": "100vw",
        }
      }
    >
      <div
        className="emotion-0"
        style={
          Object {
            "WebkitAlignItems": "center",
            "WebkitBoxAlign": "center",
            "WebkitBoxPack": "center",
            "WebkitJustifyContent": "center",
            "alignItems": "center",
            "backgroundColor": "rgba(255,255,255,1.00)",
            "borderBottomLeftRadius": "100px",
            "borderBottomRightRadius": "100px",
            "borderTopLeftRadius": "100px",
            "borderTopRightRadius": "100px",
            "boxShadow": "0px 5px 30px 10px rgba(255,255,255,1)",
            "justifyContent": "center",
            "maxWidth": "800px",
            "msFlexAlign": "center",
            "msFlexPack": "center",
            "paddingBottom": "10px",
            "paddingLeft": "10px",
            "paddingRight": "10px",
            "paddingTop": "10px",
          }
        }
      >
        <div
          className="emotion-4"
        >
          <div
            className="emotion-0"
            style={
              Object {
                "WebkitAlignItems": "center",
                "WebkitBoxAlign": "center",
                "WebkitBoxPack": "center",
                "WebkitJustifyContent": "center",
                "alignItems": "center",
                "justifyContent": "center",
                "msFlexAlign": "center",
                "msFlexPack": "center",
              }
            }
          >
            <h2
              aria-level="2"
              className="emotion-6 emotion-7"
              dir="auto"
              role="heading"
              style={
                Object {
                  "color": "rgba(17,18,20,1.00)",
                  "fontFamily": "GT-Alpina, serif",
                  "fontSize": "36px",
                  "lineHeight": "40px",
                  "marginBottom": "20px",
                  "textAlign": "center",
                  "textRendering": "geometricPrecision",
                  "textTransform": "none",
                }
              }
            >
              Work with value
            </h2>
            <h4
              aria-level="4"
              className="emotion-6 emotion-7"
              dir="auto"
              role="heading"
              style={
                Object {
                  "color": "rgba(17,18,20,1.00)",
                  "fontFamily": "Inter, sans-serif",
                  "fontSize": "24px",
                  "lineHeight": "32px",
                  "marginBottom": "20px",
                  "textAlign": "center",
                  "textRendering": "geometricPrecision",
                  "textTransform": "none",
                }
              }
            >
              Join us to create money with meaning
            </h4>
          </div>
        </div>
      </div>
      <div
        className="emotion-0"
        style={
          Object {
            "backgroundColor": "rgba(255,255,255,1.00)",
            "borderBottomLeftRadius": "100px",
            "borderBottomRightRadius": "100px",
            "borderTopLeftRadius": "100px",
            "borderTopRightRadius": "100px",
            "boxShadow": "0px 5px 30px 10px rgba(255,255,255,1)",
            "paddingBottom": "10px",
            "paddingLeft": "10px",
            "paddingRight": "10px",
            "paddingTop": "10px",
          }
        }
      >
        <div
          className="emotion-4"
        >
          <div
            className="emotion-0"
            onClick={[Function]}
            onMouseDown={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            style={
              Object {
                "WebkitAlignItems": "center",
                "WebkitBoxAlign": "center",
                "alignItems": "center",
                "msFlexAlign": "center",
              }
            }
          >
            <a
              className="emotion-6 emotion-14 emotion-7"
              dir="auto"
              href="https://clabs.co/careers"
              onClick={[Function]}
              onMouseEnter={[Function]}
              rel="noopener"
              role="link"
              style={
                Object {
                  "WebkitAlignItems": "center",
                  "WebkitBoxAlign": "center",
                  "WebkitBoxFlex": 0,
                  "WebkitBoxPack": "center",
                  "WebkitFlexGrow": 0,
                  "WebkitJustifyContent": "center",
                  "alignItems": "center",
                  "backgroundColor": "rgba(53,208,127,1.00)",
                  "borderBottomColor": "rgba(53,208,127,1.00)",
                  "borderBottomLeftRadius": "3px",
                  "borderBottomRightRadius": "3px",
                  "borderBottomWidth": "1px",
                  "borderLeftColor": "rgba(53,208,127,1.00)",
                  "borderLeftWidth": "1px",
                  "borderRightColor": "rgba(53,208,127,1.00)",
                  "borderRightWidth": "1px",
                  "borderTopColor": "rgba(53,208,127,1.00)",
                  "borderTopLeftRadius": "3px",
                  "borderTopRightRadius": "3px",
                  "borderTopWidth": "1px",
                  "color": "rgba(255,255,255,1.00)",
                  "cursor": "pointer",
                  "display": "inline-flex",
                  "flexGrow": 0,
                  "fontFamily": "Jost, futura-pt, futura, sans-serif",
                  "fontSize": "20px",
                  "fontWeight": "500",
                  "justifyContent": "center",
                  "lineHeight": "16px",
                  "minWidth": "200px",
                  "msFlexAlign": "center",
                  "msFlexPack": "center",
                  "msFlexPositive": 0,
                  "opacity": 1,
                  "paddingBottom": "20px",
                  "paddingLeft": "30px",
                  "paddingRight": "30px",
                  "paddingTop": "20px",
                  "textAlign": "center",
                  "textRendering": "geometricPrecision",
                  "width": "fit-content",
                }
              }
              target="_blank"
            >
              cLabs Open Roles
            </a>
          </div>
        </div>
        <div
          className="emotion-4"
        >
          <span
            className=" emotion-17"
            style={Object {}}
          />
        </div>
        <div
          className="emotion-4"
        >
          <div
            className="emotion-0"
            onClick={[Function]}
            onMouseDown={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            style={
              Object {
                "WebkitAlignItems": "center",
                "WebkitBoxAlign": "center",
                "alignItems": "center",
                "msFlexAlign": "center",
              }
            }
          >
            <a
              className="emotion-6 emotion-14 emotion-7"
              dir="auto"
              href="https://valoraapp.com/careers"
              onClick={[Function]}
              onMouseEnter={[Function]}
              rel="noopener"
              role="link"
              style={
                Object {
                  "WebkitAlignItems": "center",
                  "WebkitBoxAlign": "center",
                  "WebkitBoxFlex": 0,
                  "WebkitBoxPack": "center",
                  "WebkitFlexGrow": 0,
                  "WebkitJustifyContent": "center",
                  "alignItems": "center",
                  "backgroundColor": "rgba(53,208,127,1.00)",
                  "borderBottomColor": "rgba(53,208,127,1.00)",
                  "borderBottomLeftRadius": "3px",
                  "borderBottomRightRadius": "3px",
                  "borderBottomWidth": "1px",
                  "borderLeftColor": "rgba(53,208,127,1.00)",
                  "borderLeftWidth": "1px",
                  "borderRightColor": "rgba(53,208,127,1.00)",
                  "borderRightWidth": "1px",
                  "borderTopColor": "rgba(53,208,127,1.00)",
                  "borderTopLeftRadius": "3px",
                  "borderTopRightRadius": "3px",
                  "borderTopWidth": "1px",
                  "color": "rgba(255,255,255,1.00)",
                  "cursor": "pointer",
                  "display": "inline-flex",
                  "flexGrow": 0,
                  "fontFamily": "Jost, futura-pt, futura, sans-serif",
                  "fontSize": "20px",
                  "fontWeight": "500",
                  "justifyContent": "center",
                  "lineHeight": "16px",
                  "minWidth": "200px",
                  "msFlexAlign": "center",
                  "msFlexPack": "center",
                  "msFlexPositive": 0,
                  "opacity": 1,
                  "paddingBottom": "20px",
                  "paddingLeft": "30px",
                  "paddingRight": "30px",
                  "paddingTop": "20px",
                  "textAlign": "center",
                  "textRendering": "geometricPrecision",
                  "width": "fit-content",
                }
              }
              target="_blank"
            >
              Valora Open Roles
            </a>
          </div>
        </div>
        <div
          className="emotion-4"
        >
          <span
            className=" emotion-17"
            style={Object {}}
          />
        </div>
        <div
          className="emotion-4"
        >
          <div
            className="emotion-0"
            onClick={[Function]}
            onMouseDown={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            style={
              Object {
                "WebkitAlignItems": "center",
                "WebkitBoxAlign": "center",
                "alignItems": "center",
                "msFlexAlign": "center",
              }
            }
          >
            <a
              className="emotion-6 emotion-14 emotion-7"
              dir="auto"
              href="https://boards.greenhouse.io/celofoundation"
              onClick={[Function]}
              onMouseEnter={[Function]}
              rel="noopener"
              role="link"
              style={
                Object {
                  "WebkitAlignItems": "center",
                  "WebkitBoxAlign": "center",
                  "WebkitBoxFlex": 0,
                  "WebkitBoxPack": "center",
                  "WebkitFlexGrow": 0,
                  "WebkitJustifyContent": "center",
                  "alignItems": "center",
                  "backgroundColor": "rgba(53,208,127,1.00)",
                  "borderBottomColor": "rgba(53,208,127,1.00)",
                  "borderBottomLeftRadius": "3px",
                  "borderBottomRightRadius": "3px",
                  "borderBottomWidth": "1px",
                  "borderLeftColor": "rgba(53,208,127,1.00)",
                  "borderLeftWidth": "1px",
                  "borderRightColor": "rgba(53,208,127,1.00)",
                  "borderRightWidth": "1px",
                  "borderTopColor": "rgba(53,208,127,1.00)",
                  "borderTopLeftRadius": "3px",
                  "borderTopRightRadius": "3px",
                  "borderTopWidth": "1px",
                  "color": "rgba(255,255,255,1.00)",
                  "cursor": "pointer",
                  "display": "inline-flex",
                  "flexGrow": 0,
                  "fontFamily": "Jost, futura-pt, futura, sans-serif",
                  "fontSize": "20px",
                  "fontWeight": "500",
                  "justifyContent": "center",
                  "lineHeight": "16px",
                  "minWidth": "200px",
                  "msFlexAlign": "center",
                  "msFlexPack": "center",
                  "msFlexPositive": 0,
                  "opacity": 1,
                  "paddingBottom": "20px",
                  "paddingLeft": "30px",
                  "paddingRight": "30px",
                  "paddingTop": "20px",
                  "textAlign": "center",
                  "textRendering": "geometricPrecision",
                  "width": "fit-content",
                }
              }
              target="_blank"
            >
              Celo Foundation Open Roles
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`JoinJobsPage without jobs renders 1`] = `
.emotion-4 {
  opacity: 0;
}

.emotion-17 {
  width: 100%;
  height: 1rem;
}

<div
  className="emotion-0"
  style={
    Object {
      "WebkitAlignItems": "center",
      "WebkitBoxAlign": "center",
      "WebkitBoxPack": "center",
      "WebkitJustifyContent": "center",
      "alignItems": "center",
      "justifyContent": "center",
      "maxWidth": "100vw",
      "msFlexAlign": "center",
      "msFlexPack": "center",
      "overflowX": "hidden",
      "overflowY": "hidden",
    }
  }
>
  <div
    className="emotion-0"
    style={
      Object {
        "height": "100vh",
        "marginBottom": "100px",
        "width": "100%",
      }
    }
  >
    <div
      className="emotion-0"
      style={
        Object {
          "WebkitAlignItems": "center",
          "WebkitBoxAlign": "center",
          "WebkitBoxPack": "center",
          "WebkitJustifyContent": "center",
          "alignItems": "center",
          "height": "100vh",
          "justifyContent": "center",
          "left": "0px",
          "msFlexAlign": "center",
          "msFlexPack": "center",
          "position": "absolute",
          "right": "0px",
          "width": "100vw",
        }
      }
    >
      <div
        className="emotion-0"
        style={
          Object {
            "WebkitAlignItems": "center",
            "WebkitBoxAlign": "center",
            "WebkitBoxPack": "center",
            "WebkitJustifyContent": "center",
            "alignItems": "center",
            "backgroundColor": "rgba(255,255,255,1.00)",
            "borderBottomLeftRadius": "100px",
            "borderBottomRightRadius": "100px",
            "borderTopLeftRadius": "100px",
            "borderTopRightRadius": "100px",
            "boxShadow": "0px 5px 30px 10px rgba(255,255,255,1)",
            "justifyContent": "center",
            "maxWidth": "800px",
            "msFlexAlign": "center",
            "msFlexPack": "center",
            "paddingBottom": "10px",
            "paddingLeft": "10px",
            "paddingRight": "10px",
            "paddingTop": "10px",
          }
        }
      >
        <div
          className="emotion-4"
        >
          <div
            className="emotion-0"
            style={
              Object {
                "WebkitAlignItems": "center",
                "WebkitBoxAlign": "center",
                "WebkitBoxPack": "center",
                "WebkitJustifyContent": "center",
                "alignItems": "center",
                "justifyContent": "center",
                "msFlexAlign": "center",
                "msFlexPack": "center",
              }
            }
          >
            <h2
              aria-level="2"
              className="emotion-6 emotion-7"
              dir="auto"
              role="heading"
              style={
                Object {
                  "color": "rgba(17,18,20,1.00)",
                  "fontFamily": "GT-Alpina, serif",
                  "fontSize": "36px",
                  "lineHeight": "40px",
                  "marginBottom": "20px",
                  "textAlign": "center",
                  "textRendering": "geometricPrecision",
                  "textTransform": "none",
                }
              }
            >
              Work with value
            </h2>
            <h4
              aria-level="4"
              className="emotion-6 emotion-7"
              dir="auto"
              role="heading"
              style={
                Object {
                  "color": "rgba(17,18,20,1.00)",
                  "fontFamily": "Inter, sans-serif",
                  "fontSize": "24px",
                  "lineHeight": "32px",
                  "marginBottom": "20px",
                  "textAlign": "center",
                  "textRendering": "geometricPrecision",
                  "textTransform": "none",
                }
              }
            >
              Join us to create money with meaning
            </h4>
          </div>
        </div>
      </div>
      <div
        className="emotion-0"
        style={
          Object {
            "backgroundColor": "rgba(255,255,255,1.00)",
            "borderBottomLeftRadius": "100px",
            "borderBottomRightRadius": "100px",
            "borderTopLeftRadius": "100px",
            "borderTopRightRadius": "100px",
            "boxShadow": "0px 5px 30px 10px rgba(255,255,255,1)",
            "paddingBottom": "10px",
            "paddingLeft": "10px",
            "paddingRight": "10px",
            "paddingTop": "10px",
          }
        }
      >
        <div
          className="emotion-4"
        >
          <div
            className="emotion-0"
            onClick={[Function]}
            onMouseDown={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            style={
              Object {
                "WebkitAlignItems": "center",
                "WebkitBoxAlign": "center",
                "alignItems": "center",
                "msFlexAlign": "center",
              }
            }
          >
            <a
              className="emotion-6 emotion-14 emotion-7"
              dir="auto"
              href="https://clabs.co/careers"
              onClick={[Function]}
              onMouseEnter={[Function]}
              rel="noopener"
              role="link"
              style={
                Object {
                  "WebkitAlignItems": "center",
                  "WebkitBoxAlign": "center",
                  "WebkitBoxFlex": 0,
                  "WebkitBoxPack": "center",
                  "WebkitFlexGrow": 0,
                  "WebkitJustifyContent": "center",
                  "alignItems": "center",
                  "backgroundColor": "rgba(53,208,127,1.00)",
                  "borderBottomColor": "rgba(53,208,127,1.00)",
                  "borderBottomLeftRadius": "3px",
                  "borderBottomRightRadius": "3px",
                  "borderBottomWidth": "1px",
                  "borderLeftColor": "rgba(53,208,127,1.00)",
                  "borderLeftWidth": "1px",
                  "borderRightColor": "rgba(53,208,127,1.00)",
                  "borderRightWidth": "1px",
                  "borderTopColor": "rgba(53,208,127,1.00)",
                  "borderTopLeftRadius": "3px",
                  "borderTopRightRadius": "3px",
                  "borderTopWidth": "1px",
                  "color": "rgba(255,255,255,1.00)",
                  "cursor": "pointer",
                  "display": "inline-flex",
                  "flexGrow": 0,
                  "fontFamily": "Jost, futura-pt, futura, sans-serif",
                  "fontSize": "20px",
                  "fontWeight": "500",
                  "justifyContent": "center",
                  "lineHeight": "16px",
                  "minWidth": "200px",
                  "msFlexAlign": "center",
                  "msFlexPack": "center",
                  "msFlexPositive": 0,
                  "opacity": 1,
                  "paddingBottom": "20px",
                  "paddingLeft": "30px",
                  "paddingRight": "30px",
                  "paddingTop": "20px",
                  "textAlign": "center",
                  "textRendering": "geometricPrecision",
                  "width": "fit-content",
                }
              }
              target="_blank"
            >
              cLabs Open Roles
            </a>
          </div>
        </div>
        <div
          className="emotion-4"
        >
          <span
            className=" emotion-17"
            style={Object {}}
          />
        </div>
        <div
          className="emotion-4"
        >
          <div
            className="emotion-0"
            onClick={[Function]}
            onMouseDown={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            style={
              Object {
                "WebkitAlignItems": "center",
                "WebkitBoxAlign": "center",
                "alignItems": "center",
                "msFlexAlign": "center",
              }
            }
          >
            <a
              className="emotion-6 emotion-14 emotion-7"
              dir="auto"
              href="https://valoraapp.com/careers"
              onClick={[Function]}
              onMouseEnter={[Function]}
              rel="noopener"
              role="link"
              style={
                Object {
                  "WebkitAlignItems": "center",
                  "WebkitBoxAlign": "center",
                  "WebkitBoxFlex": 0,
                  "WebkitBoxPack": "center",
                  "WebkitFlexGrow": 0,
                  "WebkitJustifyContent": "center",
                  "alignItems": "center",
                  "backgroundColor": "rgba(53,208,127,1.00)",
                  "borderBottomColor": "rgba(53,208,127,1.00)",
                  "borderBottomLeftRadius": "3px",
                  "borderBottomRightRadius": "3px",
                  "borderBottomWidth": "1px",
                  "borderLeftColor": "rgba(53,208,127,1.00)",
                  "borderLeftWidth": "1px",
                  "borderRightColor": "rgba(53,208,127,1.00)",
                  "borderRightWidth": "1px",
                  "borderTopColor": "rgba(53,208,127,1.00)",
                  "borderTopLeftRadius": "3px",
                  "borderTopRightRadius": "3px",
                  "borderTopWidth": "1px",
                  "color": "rgba(255,255,255,1.00)",
                  "cursor": "pointer",
                  "display": "inline-flex",
                  "flexGrow": 0,
                  "fontFamily": "Jost, futura-pt, futura, sans-serif",
                  "fontSize": "20px",
                  "fontWeight": "500",
                  "justifyContent": "center",
                  "lineHeight": "16px",
                  "minWidth": "200px",
                  "msFlexAlign": "center",
                  "msFlexPack": "center",
                  "msFlexPositive": 0,
                  "opacity": 1,
                  "paddingBottom": "20px",
                  "paddingLeft": "30px",
                  "paddingRight": "30px",
                  "paddingTop": "20px",
                  "textAlign": "center",
                  "textRendering": "geometricPrecision",
                  "width": "fit-content",
                }
              }
              target="_blank"
            >
              Valora Open Roles
            </a>
          </div>
        </div>
        <div
          className="emotion-4"
        >
          <span
            className=" emotion-17"
            style={Object {}}
          />
        </div>
        <div
          className="emotion-4"
        >
          <div
            className="emotion-0"
            onClick={[Function]}
            onMouseDown={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            style={
              Object {
                "WebkitAlignItems": "center",
                "WebkitBoxAlign": "center",
                "alignItems": "center",
                "msFlexAlign": "center",
              }
            }
          >
            <a
              className="emotion-6 emotion-14 emotion-7"
              dir="auto"
              href="https://boards.greenhouse.io/celofoundation"
              onClick={[Function]}
              onMouseEnter={[Function]}
              rel="noopener"
              role="link"
              style={
                Object {
                  "WebkitAlignItems": "center",
                  "WebkitBoxAlign": "center",
                  "WebkitBoxFlex": 0,
                  "WebkitBoxPack": "center",
                  "WebkitFlexGrow": 0,
                  "WebkitJustifyContent": "center",
                  "alignItems": "center",
                  "backgroundColor": "rgba(53,208,127,1.00)",
                  "borderBottomColor": "rgba(53,208,127,1.00)",
                  "borderBottomLeftRadius": "3px",
                  "borderBottomRightRadius": "3px",
                  "borderBottomWidth": "1px",
                  "borderLeftColor": "rgba(53,208,127,1.00)",
                  "borderLeftWidth": "1px",
                  "borderRightColor": "rgba(53,208,127,1.00)",
                  "borderRightWidth": "1px",
                  "borderTopColor": "rgba(53,208,127,1.00)",
                  "borderTopLeftRadius": "3px",
                  "borderTopRightRadius": "3px",
                  "borderTopWidth": "1px",
                  "color": "rgba(255,255,255,1.00)",
                  "cursor": "pointer",
                  "display": "inline-flex",
                  "flexGrow": 0,
                  "fontFamily": "Jost, futura-pt, futura, sans-serif",
                  "fontSize": "20px",
                  "fontWeight": "500",
                  "justifyContent": "center",
                  "lineHeight": "16px",
                  "minWidth": "200px",
                  "msFlexAlign": "center",
                  "msFlexPack": "center",
                  "msFlexPositive": 0,
                  "opacity": 1,
                  "paddingBottom": "20px",
                  "paddingLeft": "30px",
                  "paddingRight": "30px",
                  "paddingTop": "20px",
                  "textAlign": "center",
                  "textRendering": "geometricPrecision",
                  "width": "fit-content",
                }
              }
              target="_blank"
            >
              Celo Foundation Open Roles
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
